reactioncommerce/reaction
# Fix price range display in product grid

This resolves #2521 

### Steps to test
1. Clone the "Basic Reaction Product"
1. Archive the "Red" option so the price just shows $12.99 in the PDP
1. Publish
1. Observe that the price range on the product grid  changes to $12.99
